### PR TITLE
Remove leftover pip.installed

### PR DESCRIPTION
**Pull request by giorgiosironi**
Follow-up to https://github.com/elifesciences/builder-base-formula/pull/186 with the bits not strictly necessary for elife-xpub.

### CODE REVIEW

File: elife/pypi.sls [hunk 1]
```diff
@@ -15,6 +15,5 @@ pypi-credentials-ubuntu-user:
             - deploy-user
 
 twine:
-    pip.installed:
-        - pkgs:
-            - twine
+    cmd.run:
+        - name: pip install twine
```
Comment (diversemix) [inline elife/pypi.sls]: Did you want to pin a version of this? (ref: https://pypi.org/project/twine/#history )

Comment (giorgiosironi) [reply elife/pypi.sls]: At the moment it doesn't pin a version, by default I'm keeping the same behavior. Perhaps pinning `1.x` would be a good idea.

**Comment by giorgiosironi** [comment]
(18.04 to be fixed later, also thanks to this changes)

File: elife/uwsgi.sls [hunk 2]
```diff
@@ -18,8 +18,8 @@ uwsgi-pkg:
 {% else %}
 # apps should be installing and using their own version of uwsgi
 uwsgi-pkg:
-    pip.installed:
-        - name: uwsgi >= 2.0.8
+    cmd.run:
+        - name: pip install "uwsgi>=2.0.8"
```
Comment (lsh-0) [inline elife/uwsgi.sls]: we should figure out what projects are still using the global uwsgi and update them.

Comment (giorgiosironi) [reply elife/uwsgi.sls]: some that have been moved to Docker containers naturally had to use their own `uwsgi` in the virtualenv

**Pull request merged**